src/test/java/ContactRefactor.java

```diff
@@ -0,0 +1,122 @@
+import org.junit.*;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.By;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.util.concurrent.TimeUnit;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+public class ContactRefactor {
+    private static ChromeDriver driver;
+    private static JavascriptExecutor js;
+    WebElement element;
+    Actions builder;
+    String urlBase = "https://jpcorp-dev-ed.lightning.force.com";
+    String username = "juampi7237@gmail.com";
+    String password = "JP123456jp";
+    String firstName = "Juan";
+    String lastName = "Gonzales";
+    WebDriverWait wait;
+
+    @BeforeClass
+    public static void setup() throws Exception {
+        System.setProperty("webdriver.chrome.driver", "./src/test/resources/drivers/chromedriver.exe");
+        driver = new ChromeDriver();
+        js = (JavascriptExecutor) driver;
+        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+        driver.manage().window().maximize();
+    }
+
+    @Before
+    public void Login() {
+        driver.get(urlBase);
+        driver.findElement(By.id("username")).sendKeys(username);
+        driver.findElement(By.id("password")).sendKeys(password);
+        driver.findElement(By.id("Login")).click();
+    }
+
+
+    @Test
+    public void createContact() {
+        wait = new WebDriverWait(driver, 30);
+        driver.findElement(By.cssSelector(".slds-icon-waffle")).click();
+        driver.findElement(By.xpath("//input[@placeholder='Search apps and items...']")).click();
+        driver.findElement(By.xpath("//input[@placeholder='Search apps and items...']")).sendKeys("Contacts");
+        WebElement element = driver.findElement(By.xpath("//a[@data-label='Contacts']"));
+        js.executeScript("arguments[0].click();", element);
+        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+        driver.findElement(By.xpath("//div[contains(text(),'New')]")).click();
+        driver.findElement(By.xpath("//input[@name='lastName']")).click();
+        driver.findElement(By.xpath("//input[@name='lastName']")).sendKeys(lastName);
```

Comment:
It should not be necesary to click on the textbox and write the value. Only with the sendkeys should work,